__tests__/app.test.js

```diff
@@ -26,29 +26,53 @@ describe("GET /api", () => {
 
 describe("GET /api/topics", () => {
   test("200: Responds with an array of topic objects", () => {
-    const topics = [
-      {
-        description: "The man, the Mitch, the legend",
-        slug: "mitch",
-      },
-      {
-        description: "Not dogs",
-        slug: "cats",
-      },
-      {
-        description: "what books are made of",
-        slug: "paper",
-      },
-    ];
-
     return request(app)
       .get("/api/topics")
       .expect(200)
       .then(({ body }) => {
-        expect(body).toEqual(topics);
+        expect(body).toHaveLength(3);
+        body.forEach((slug) => {
+          expect(slug).toMatchObject({
+            description: expect.any(String),
+            slug: expect.any(String),
+          });
+        });
+      });
+  });
+  test("should return 404 for an invalid url", () => {
+    return request(app)
+      .get("/api/topic") // Missing s
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Not Found");
```

Comment:
I would put this test into it's own describe block and make the path more distinctive - /api/topic is too similar. The test should still pass if you change it to /api/notaroute for example